modules/nw-ingress-controller-nodeportservice-projects.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/nw-configuring-ingress-controller-endpoint-publishing-strategy.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-controller-nodeportservice-projects_{context}"]
+= Adding a single NodePort service to an Ingress Controller
+
+Instead of creating a `NodePort`-type `Service` for each project, you can create a single service and apply it to all of your projects. You can configure this type of environment by setting your Ingress Controller to use the `NodePortService` endpoint publishing strategy.
+
+Consider this configuration for your Ingress Controller when you want to apply a set of routes, through Ingress sharding, to a node even if the node gets displaced. 
```

Comment:
```suggestion
Consider this configuration for your Ingress Controller when you want to apply a set of routes, through Ingress sharding, to nodes that may already have a HostNetwork Ingress Controller because there will not be any port conflicts.  
```